app/_includes/md/konnect/create-service.md

```diff
@@ -0,0 +1,22 @@
+<!-- Used in Konnect getting started guides -->
+
+1. From the left navigation menu, click **Services** to open ServiceHub.
+
+1. Click **Add New Service**.
+
+1. Enter a **Service Name**. For this example, enter `example_service`.
+
+    A Service name can be any string containing letters, numbers, or characters;
+    for example, `service_name`, `Service Name`, or `Service-name`.
```

Comment:
New as of this weekend. Due to auto-discovery implementation. The Service name string is constrained by gateway tags.
```
^[0-9a-zA-Z.\-_~:]*$
```
The notable constraint: no spaces.

Note: We'll look to relax this limitation post helium. But does not seem likely before.

Reply:
This doesn't seem to be the case for Konnect Services in the ServiceHub, which is what this topic is describing. The error only happens when you create a Gateway service.

Reply:
If you try implement a Service Version after naming a Konnect Service with an invalid name (eg: `Service A`), the SV will fail to be created. We will catch this error properly soon at the Konnect Service level.

Reply:
Fixed in https://github.com/Kong/docs.konghq.com/pull/3768/commits/76a4a2685dc8f77d388622d9825a7b8a21a5506d.